libs/openFrameworks/video/ofAVFoundationVideoPlayer.h

```diff
@@ -42,7 +42,7 @@ typedef enum _playerLoopType{
 	AVAssetReaderTrackOutput * _assetReaderVideoTrackOutput;
 	AVAssetReaderTrackOutput * _assetReaderAudioTrackOutput;
 	
-#if USE_VIDEO_OUTPUT
+#if defined(USE_VIDEO_OUTPUT)
```

Comment:
This define did not resolve unambiguously.  Using `defined(...)` removes the ambiguity.